main/test/flix/Test.Exp.Regions.flix

```diff
@@ -454,7 +454,7 @@ mod Test.Exp.Regions {
                 MutList.push!(1, l);
                 MutMap.put!(1, l, Array.get(0, a));
                 match MutMap.get(1, Array.get(0, a)) {
-                    case Some(k) => MutList.nth(0, k) == Some(1)
+                    case Some(k) => optionalNth(0, k) == Some(1)
```

Comment:
These are just test cases, so I think we should just use `get` and avoid all the optionalNth stuff!

Reply:
There is no `get` in `MutList`. Should I: 
1. add it and delegate down to `Array.get`
2. use `nth` and put try..with statements around the relevant tests?